src/commands/sns_canisters_summary.rs

```diff
@@ -0,0 +1,39 @@
+use crate::lib::signing::{sign_ingress_with_request_status_query, IngressWithRequestId};
+use crate::lib::{AnyhowResult, TargetCanister};
+use crate::SnsCanisterIds;
+use candid::Encode;
+use clap::Parser;
+use ic_base_types::{CanisterId, PrincipalId};
+use std::str::FromStr;
+
+#[derive(Parser, Debug)]
+#[clap(about, version, author)]
```

Comment:
This probably should go in mod.rs on the top level command instead on only some of the sub commands. If you feel strongly about it, please move it there. If not please remove it.